pyproject.toml

```
@@ -12,9 +12,37 @@ python = "^3.10"
 requests = "^2.28.1"
 
 [tool.poetry.group.dev.dependencies]
-pytest = "^7.1.3"
-responses = "^0.21.0"
-black = "^22.8.0"
+pytest = "^7.2.0"
+responses = "^0.22.0"
+black = "^22.10.0"
+flake8 = "^5.0.4"
+flake8-bugbear = "^22.10.27"
+flake8-builtins = "^2.0.1"
+flake8-comprehensions = "^3.10.1"
+flake8-debugger = "^4.1.2"
+flake8-unused-arguments = "^0.0.12"
+flake8-quotes = "^3.3.1"
+flake8-print = "^5.0.0"
+flake8-variables-names = "^0.0.5"
+pep8-naming = "^0.13.2"
+flake8-pyprojecttoml = "^0.0.2"
+flake8-assertive = "^2.1.0"
+isort = "^5.10.1"
+flake8-class-attributes-order = "^0.1.3"
+mypy = "^0.991"
+types-requests = "^2.28.11.4"
+
+[tool.flake8]
+max_line_length = 120
+inline-quotes = "double"
+# Line below ignores a bug. See https://github.com/PyCQA/pycodestyle/issues/373
+extend-ignore = "E203"
+
+[tool.isort]
+line_length = 120
```

```suggestion
[tool.isort]
profile = "black"
```
`isort` seems to be suggesting above: https://pycqa.github.io/isort/docs/configuration/black_compatibility.html